### PR TITLE
[IMP] l10n_ua: update CoA & taxes and translate l10n to English

### PR DESCRIPTION
**Pull request by sovi-odoo**
- Apply changes suggested by https://github.com/odoo/odoo/pull/133282
- Translate the l10n to English (there may be some mistakes as I don't speak Ukrainian)
- Add "Cash" accounts (301, 302, 308, 309) as described here: https://zakon.rada.gov.ua/laws/show/z0893-99
- Remove useless taxes & unused IAS CoA
- Correct a few typos

Closes https://github.com/odoo/odoo/pull/133282
task-3790513

Co-Authored-By: Bohdan Lisnenko <lem8r.ka11a@gmail.com>

### CODE REVIEW

**Review by malb-odoo — COMMENTED** [review]
Commit message is a bit wrong :smile: , when it's a task you should not put opw but task 
image

File: addons/l10n_ua/data/template/account.account-ua_ias.csv [hunk 1]
```diff
@@ -1,77 +1,77 @@
...
+"ua_ias_1170","1170","Long-term accounts receivable","asset_non_current","False","","Довгострокова дебіторська заборгованість"
+"ua_ias_1180","1180","Deferred tax assets","asset_non_current","False","","Відстрочені податкові активи"
+"ua_ias_1200","1200","Accounts payable for goods and services","liability_payable","True","","Кредиторська заборгованість за товари та послуги"
+"ua_ias_1201","1201","Received to warehouse without invoice","liability_current","True","","Отримано на склад без рахунку-фактури"
+"ua_ias_1202","1202","Income tax","liability_payable","True","","Податок на прибуток"
+"ua_ias_1203","1203","VAT","liability_payable","True","l10n_ua.acc_tag_vat","ПДВ"
+"ua_ias_1204","1204","VAT liabilities","liability_current","False","","Податкові зобов’язання по ПДВ"
+"ua_ias_1205","1205","Personal income tax","liability_payable","True","","ПДФО"
+"ua_ias_12051","12051","Military fee","liability_payable","True","","Військовий збір"
+"ua_ias_1206","1206","ЄСВ","liability_payable","True","","ЄСВ"
```
Comment (malb-odoo) [inline addons/l10n_ua/data/template/account.account-ua_ias.csv:43]: one missing trad ?

File: addons/l10n_ua/data/template/account.tax-ua_ias.csv [hunk 2]
```diff
@@ -1,81 +1,74 @@
...
-"simple_tax_sale_15","33","Дохід, що оподатковується за ставкою 15%","","Дохід за ставкою 15%","0.0","sale","","","","",""
-"simple_tax_purchase_product","40","Витрати від продажу товарів","","Витрати від продажу товарів","0.0","purchase","","","","",""
-"simple_tax_purchase_salary","41","Витрати на оплату праці найманих працівників","","Витрати на оплату праці найманих працівників","0.0","purchase","","","","",""
-"simple_tax_purchase_esv","42","Витрати на ЄСВ","","Витрати на ЄСВ","0.0","purchase","","","","",""
-"simple_tax_purchase_other","43","Витрати інші","","Витрати інші","0.0","purchase","","","","",""
+"id","sequence","name","description","invoice_label","amount","type_tax_use","tax_group_id","active","price_include","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/account_id","name@uk_UA","invoice_label@uk_UA"
+"sale_tax_template_vat20","9","20%","","20%","20.0","sale","tax_group_vat20","","","base","invoice","","Реалізація з ПДВ 20%","+ ПДВ 20%"
```
Comment (malb-odoo) [inline addons/l10n_ua/data/template/account.tax-ua_ias.csv:2]: Just to be sure the invoice label is completely changed compared to the original one, do we want that ?

File: addons/l10n_ua/data/template/account.tax-ua_ias.csv [hunk 3]
```diff
@@ -1,81 +1,74 @@
...
+"","","","","","","","","","","tax","refund","ua_ias_1204","",""
+"sale_tax_template_vat7","11","7%","","7%","7.0","sale","tax_group_vat7","","","base","invoice","","Реалізація з ПДВ 7%","+ ПДВ 7%"
+"","","","","","","","","","","tax","invoice","ua_ias_1204","",""
+"","","","","","","","","","","base","refund","","",""
+"","","","","","","","","","","tax","refund","ua_ias_1204","",""
+"sale_tax_template_vat7incl","11","7% INC","","7% INC","7.0","sale","tax_group_vat7","False","True","base","invoice","","Реалізація в т. ч. ПДВ 7%","в т .ч. ПДВ 7%"
+"","","","","","","","","","","tax","invoice","ua_ias_1204","",""
+"","","","","","","","","","","base","refund","","",""
+"","","","","","","","","","","tax","refund","ua_ias_1204","",""
+"sale_tax_template_vat0","12","0%","","0%","0.0","sale","tax_group_vat0","","","base","invoice","","Реалізація з ПДВ 0%","ПДВ 0%"
```
Comment (malb-odoo) [inline addons/l10n_ua/data/template/account.tax-ua_ias.csv:26]: The tax group 0 was added, seems logic but just to be sure we want that

File: addons/l10n_ua/data/template/account.tax-ua_ias.csv [hunk 4]
```diff
@@ -1,81 +1,74 @@
...
+"","","","","","","","","","","tax","refund","ua_ias_1140","",""
+"purchase_tax_template_vat7incl","21","7% INC","","7% INC","7.0","purchase","tax_group_vat7","False","True","base","invoice","","Придбання в т. ч. ПДВ 7%","в т. ч. ПДВ 7%"
+"","","","","","","","","","","tax","invoice","ua_ias_1140","",""
+"","","","","","","","","","","base","refund","","",""
+"","","","","","","","","","","tax","refund","ua_ias_1140","",""
+"purchase_tax_template_vat0","22","0%","","0%","0.0","purchase","tax_group_vat0","","","base","invoice","","Придбання з ПДВ 0%","ПДВ 0%"
+"","","","","","","","","","","tax","invoice","","",""
+"","","","","","","","","","","base","refund","","",""
+"","","","","","","","","","","tax","refund","","",""
+"purchase_tax_template_vat_free","23","VAT Exempt","","VAT Exempt","0.0","purchase","tax_group_vat0","","","base","invoice","","Придбання звільнене від ПДВ","Звільнено від ПДВ"
```
Comment (malb-odoo) [inline addons/l10n_ua/data/template/account.tax-ua_ias.csv:66]: Name should be 0% Exempt

File: addons/l10n_ua/data/template/account.tax-ua_ias.csv [hunk 5]
```diff
@@ -1,81 +1,74 @@
...
+"","","","","","","","","","","tax","invoice","","",""
+"","","","","","","","","","","base","refund","","",""
+"","","","","","","","","","","tax","refund","","",""
+"purchase_tax_template_vat_not","24","VAT NS","","VAT NS","0.0","purchase","tax_group_vat0","","","base","invoice","","Придбання не є об'єктом ПДВ","Не є об'єктом ПДВ"
```
Comment (malb-odoo) [inline addons/l10n_ua/data/template/account.tax-ua_ias.csv:70]: Name should be 0% NS, Why NS ?

File: addons/l10n_ua/data/template/account.tax-ua_ias.csv [hunk 6]
```diff
@@ -1,81 +1,74 @@
...
-"","","","","","","","","","base","refund",""
-"","","","","","","","","","tax","refund",""
-"simple_tax_sale_product","30","Дохід від продажу товарів","","Дохід від продажу товарів","0.0","sale","","","","",""
-"simple_tax_sale_gift","31","Дохід від безоплатно отриманих товарів","","Дохід від безоплатно отриманих товарів","0.0","sale","","","","",""
-"simple_tax_sale_old","32","Дохід від заборгованності, за якою минув строк позивної давності","","Дохід від заборгованності, за якою минув строк позивної давності","0.0","sale","","","","",""
-"simple_tax_sale_15","33","Дохід, що оподатковується за ставкою 15%","","Дохід за ставкою 15%","0.0","sale","","","","",""
-"simple_tax_purchase_product","40","Витрати від продажу товарів","","Витрати від продажу товарів","0.0","purchase","","","","",""
-"simple_tax_purchase_salary","41","Витрати на оплату праці найманих працівників","","Витрати на оплату праці найманих працівників","0.0","purchase","","","","",""
-"simple_tax_purchase_esv","42","Витрати на ЄСВ","","Витрати на ЄСВ","0.0","purchase","","","","",""
-"simple_tax_purchase_other","43","Витрати інші","","Витрати інші","0.0","purchase","","","","",""
```
Comment (malb-odoo) [inline addons/l10n_ua/data/template/account.tax-ua_ias.csv:81]: It's missing some of those, Forgot to add or on purpose ?

File: addons/l10n_ua/data/template/account.tax-ua_psbo.csv [hunk 7]
```diff
@@ -1,81 +1,74 @@
-"id","sequence","name","description","invoice_label","amount","type_tax_use","tax_group_id","price_include","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/account_id"
```
Comment (malb-odoo) [inline addons/l10n_ua/data/template/account.tax-ua_psbo.csv:1]: Comments apply to this file also

File: addons/l10n_ua/data/template/account.tax.group-ua_ias.csv [hunk 8]
```diff
@@ -1,7 +1,8 @@
...
-"tax_group_vat0","ПДВ 0%","base.ua"
-"tax_group_vat_free","Звільнено від ПДВ","base.ua"
-"tax_group_not_vat","Не є ПДВ","base.ua"
+"id","name","country_id","name@uk_UA"
+"tax_group_vat20","VAT 20%","base.ua","ПДВ 20%"
+"tax_group_vat15","VAT 15%","base.ua","ПДВ 15%"
+"tax_group_vat14","VAT 14%","base.ua","ПДВ 14%"
+"tax_group_vat7","VAT 7%","base.ua","ПДВ 7%"
+"tax_group_vat0","VAT 0%","base.ua","ПДВ 0%"
+"tax_group_vat_free","Exempt from VAT","base.ua","Звільнено від ПДВ"
```
Comment (malb-odoo) [inline addons/l10n_ua/data/template/account.tax.group-ua_ias.csv:7]: 0% Exempt

File: addons/l10n_ua/data/template/account.tax.group-ua_ias.csv [hunk 9]
```diff
@@ -1,7 +1,8 @@
...
+"tax_group_not_vat","VAT not applicable","base.ua","Не є ПДВ"
```
Comment (malb-odoo) [inline addons/l10n_ua/data/template/account.tax.group-ua_ias.csv:8]: 0% No Vat ?

File: addons/l10n_ua/data/template/account.tax.group-ua_psbo.csv [hunk 10]
```diff
@@ -1,7 +1,8 @@
...
-"tax_group_vat0","ПДВ 0%","base.ua"
-"tax_group_vat_free","Звільнено від ПДВ","base.ua"
-"tax_group_not_vat","Не є ПДВ","base.ua"
+"id","name","country_id","name@uk_UA"
+"tax_group_vat20","VAT 20%","base.ua","ПДВ 20%"
+"tax_group_vat15","VAT 15%","base.ua","ПДВ 15%"
+"tax_group_vat14","VAT 14%","base.ua","ПДВ 14%"
+"tax_group_vat7","VAT 7%","base.ua","ПДВ 7%"
+"tax_group_vat0","VAT 0%","base.ua","ПДВ 0%"
+"tax_group_vat_free","Exempt from VAT","base.ua","Звільнено від ПДВ"
```
Comment (malb-odoo) [inline addons/l10n_ua/data/template/account.tax.group-ua_psbo.csv:7]: same here and for the other

**Review by malb-odoo — CHANGES_REQUESTED** [review]
Thanks for this :smile: , some comments and questions ?

File: addons/l10n_ua/data/template/account.tax-ua_ias.csv (hunk 5 above)
Comment (sovi-odoo) [reply addons/l10n_ua/data/template/account.tax-ua_ias.csv:70]: Shorthand for "Not Subject". It comes from l10n_es. You can see an example here: https://github.com/odoo/odoo/blob/a773c1d289e74a788bab2049fee226a3eb2db430/addons/l10n_es/data/template/account.tax-es_common.csv#L124

If "NS" isn't clear, would it be better to write it as "0% Not Subject" instead?

**Review by malb-odoo — APPROVED** [review]
LGTM good job on this, keep up the good work :smile:

**Review by qdp-odoo — CHANGES_REQUESTED** [review]
@sovi-odoo thank you, it's almost good for me: can you just grant the authorship (or co-authorship) to @lem8r since he came with that branch at first?

thanks :+1:

**Comment by lem8r** [comment]
It would be great to drop IAS chart template completely as it is not used anywhere for real.
National CoA can be used for IFRS and it is used in real life as it is more convenient.

**Comment by sovi-odoo** [comment]
IAS removed. Should be good to merge now :)

**Comment by qdp-odoo** [comment]
@robodoo r+

**Pull request closed without merging**